src/components/AppMap/AppMap.vue

```diff
@@ -44,21 +48,23 @@
 import { mapState } from "vuex";
 import { bbox } from "@turf/turf";
 import MapLegend from "@/components/MapboxMap/MapLegend.vue";
-import MapControlFitbounds from "@/components/MapboxMap/MapControlFitbounds.vue";
+import MapFitboundsControl from "@/components/MapboxMap/MapFitboundsControl.vue";
+import MapStyleControl from "@/components/MapboxMap/MapStyleControl.vue";
 import MapDrawControl from "@/components/MapboxMap/MapDrawControl.vue";
 
 export default {
   components: {
-    MapControlFitbounds,
+    MapFitboundsControl,
     MapLegend,
     MapDrawControl,
+    MapStyleControl,
   },
 
   data() {
     return {
       mapConfig: {
         accessToken: process.env.VUE_APP_MAPBOX_TOKEN,
-        style: "mapbox://styles/siggyf/ckww2c33f0xlf15nujlx41fe2",
+        style: "mapbox://styles/voorhoede/clh8u52h600wh01pn73ns8zy8",
```

Comment:
I made a copy of Fedor's style and uploaded it to our studio mapbox, that way we only need one `VUE_APP_MAPBOX_TOKEN`